spring-cloud-gcp-data-datastore/src/test/java/com/google/cloud/spring/data/datastore/it/TestEntityRepository.java

```diff
@@ -134,6 +135,11 @@
 
 	Optional<TestEntity> findFirstByColor(String color);
 
+	Stream<TestEntity> findStreamByColor(String color);
+
+	@Query("select * from  test_entities_ci where color = @color")
+	Stream<TestEntity> findByColorStream(@Param("color") String color);
```

Comment:
The difference between `findStreamByColor` and `findByColorStream` is very subtle. Could you rename `findByColorStream` to something like `findGqlStreamByColor`? 

And possibly rename `findStreamByColor` to `findPartTreeStreamByColor`, so the targeted functionality is very clear.

Reply:
hmm, do you think I should also make a comment here say these method are to test Stream as return type for GQL query and PartTree query respectively? Or is it too much?

Reply:
no, I think the descriptive naming you have right now is great.